books.js

```diff
@@ -1,62 +1,66 @@
 /* eslint-disable linebreak-style */
-/* eslint-disable no-use-before-define */
-/* eslint-disable no-unused-vars */
-const addBtn = document.getElementsByClassName('add-btn')[0];
-const bookSection = document.getElementsByClassName('books')[0];
-let books = [];
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
```

Comment:
- [ ] You said that you are allowed to submit this activity without your partner commits. Can attach a screenshot of the email(In which you got permission from StudentSuccessTeam) in your pull request :blush: